website/project/views/contributor.py

```diff
@@ -378,6 +378,55 @@ def project_manage_contributors(auth, node, **kwargs):
     return {}
 
 
+@no_auto_transaction
+@must_be_valid_project  # injects project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def project_remove_contributor(auth, **kwargs):
+    """Remove a contributor from a list of nodes.
+
+    :param Auth auth: Consolidated authorization
+    :param-json list contributors: Ordered list of contributors represented as
+        dictionaries of the form:
+        {'id': <id>, 'permission': <One of 'read', 'write', 'admin'>}
```

Comment:
Is this param accurate? It looks like a hold-over from [`project_manage_contributors`](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/project/views/contributor.py#L343-L345), and I'm not sure it's even accurate over there.